do/csi/driver/driver_test.go

```diff
@@ -61,9 +55,15 @@ func TestDriverSuite(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	mntStageDir, err := ioutil.TempDir("", "mnt-stage")
```

Comment:
Should we defer a directory delete

Reply:
@cagedmantis I'll do a follow up PR with this fix included.